storage/migrations/mysql/1_init.sql

```diff
@@ -166,16 +167,18 @@ CREATE TABLE IF NOT EXISTS `deliveries` (
   INDEX id_created_at (`id`, `created_at`)
 ) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 
-CREATE TABLE IF NOT EXISTS `send_bulk_logs` (
-  `id`          bigint unsigned primary key AUTO_INCREMENT NOT NULL,
-  `uuid`        varchar(36) NOT NULL,
-  `user_id`     integer unsigned NOT NULL,
-  `campaign_id` integer unsigned NOT NULL,
-  `message_id`  varchar(191) NOT NULL,
-  `status`      varchar(191) NOT NULL,
-  `created_at`  datetime(6) NOT NULL,
+CREATE TABLE IF NOT EXISTS `send_logs` (
+  `id`              bigint unsigned primary key AUTO_INCREMENT NOT NULL,
+  `uuid`            varchar(36) NOT NULL,
+  `user_id`         integer unsigned NOT NULL,
+  `subscriber_id`   integer unsigned NOT NULL,
+  `campaign_id`     integer unsigned NOT NULL,
+  `status`          varchar(191) NOT NULL,
+  `description`     varchar(191) NOT NULL,
+  `created_at`      datetime(6) NOT NULL,
   FOREIGN KEY (`user_id`) REFERENCES users(`id`),
-  FOREIGN KEY (`campaign_id`) REFERENCES campaigns(`id`)
+  FOREIGN KEY (`campaign_id`) REFERENCES campaigns(`id`),
+  INDEX u_s_c_id (`user_id`, `uuid`,`campaign_id`)
```

Comment:
I'm not too sure we need this index, we need to see what our reads are first since an index can be costly especially in a table with a high number of writes.